index.js

```diff
@@ -43,16 +43,16 @@ function checkGuess() {
     if (guess < targetNumber) {
       tooLowMessage.style.display = '';
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = '';
     }
 
-    const remainingAttempts = maxNumberOfAttempts - attempts;
+    let remainingAttempts = maxNumberOfAttempts - attempts;
```

Comment:
Nitpick: The `remainingAttempts` variable can also be a `const` variable because while its value changes each time a new guess is submitted, it's value is never changed below line 49 and thrown away after line 63.

For example:
- You've submitted a guess
- This variable is calculated as `5 (maxNumberOfAttempts) - 1 (attempts) = 4`;
- After checkGuess is done running, this `remainingAttempts` value is thrown away
- When you submit another guess
- This variable is recalculated as `5 (maxNumberOfAttempts) - 2 (attempts) = 3`;
- After checkGuess is done running, this `remainingAttempts` value is thrown away
- ...etc

You can think of this variable as a **temporary** value that is thrown away after the `checkGuess`'s closing bracket `}` on line 63 is read by the computer. Therefore, this can be changed to
`const remainingAttempts = maxNumberOfAttempts - attempts;` 
because its value is always recalculated and thrown away for each guess!